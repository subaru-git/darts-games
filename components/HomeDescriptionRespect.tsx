import React, { FC } from 'react'
import { Center, Grid, GridItem, Text } from '@chakra-ui/react'
import YouTube from 'react-youtube'

const HomeDescriptionRespect: FC = () => {
  return (
    <div>
      <Grid templateColumns='repeat(2, auto)' justifyContent='center' gap={4}>
        <GridItem>
          <Center h='100%'>
            <Text fontSize='xl'>
              This application respects all darts content like youtube videos, blogs, and more.
            </Text>
          </Center>
        </GridItem>
        <GridItem>
          <YouTube videoId='HiNOjEYbUXc' />
        </GridItem>
      </Grid>
    </div>
  )
}

export default HomeDescriptionRespect
