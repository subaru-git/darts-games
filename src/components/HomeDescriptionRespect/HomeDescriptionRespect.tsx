import React, { FC } from 'react'
import { Center, Grid, GridItem, Text } from '@chakra-ui/react'
import YouTube from 'react-youtube'
import useLocale from '@/hooks/locale'

const HomeDescriptionRespect: FC = () => {
  const { t } = useLocale()
  return (
    <div>
      <Grid templateColumns='repeat(2, auto)' justifyContent='center' gap={4}>
        <GridItem>
          <Center h='100%'>
            <Text fontSize='xl'>{t.HOME_RESPECT}</Text>
          </Center>
        </GridItem>
        <GridItem>
          <YouTube videoId='HiNOjEYbUXc' />
        </GridItem>
      </Grid>
    </div>
  )
}

export default HomeDescriptionRespect
