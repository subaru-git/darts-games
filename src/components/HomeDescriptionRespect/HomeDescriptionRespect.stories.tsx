/* eslint-disable import/named */
import React from 'react'
import { ComponentMeta, ComponentStory } from '@storybook/react'
import HomeDescriptionRespect from './HomeDescriptionRespect'

export default {
  title: 'Components/HomeDescriptionRespect',
  component: HomeDescriptionRespect,
} as ComponentMeta<typeof HomeDescriptionRespect>

const Template: ComponentStory<typeof HomeDescriptionRespect> = (args) => (
  <HomeDescriptionRespect {...args} />
)

export const Default = Template.bind({})
Default.args = {}
