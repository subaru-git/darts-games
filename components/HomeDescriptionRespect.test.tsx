import { render } from '@testing-library/react'
import HomeDescriptionRespect from './HomeDescriptionRespect'

test('should rendering', () => {
  const { container } = render(<HomeDescriptionRespect />)
  expect(container.getElementsByClassName('YouTube')).not.toBeNull()
  expect(container).toMatchSnapshot()
})
