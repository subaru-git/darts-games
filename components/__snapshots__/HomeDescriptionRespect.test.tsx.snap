// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should rendering 1`] = `
<div>
  <div>
    <div
      class="css-0"
    >
      <div
        class="css-0"
      >
        <div
          class="css-0"
        >
          <p
            class="chakra-text css-0"
          >
            This application respects all darts content like youtube videos, blogs, and more.
          </p>
        </div>
      </div>
      <div
        class="css-0"
      >
        <div
          class=""
        >
          <div
            class=""
            id=""
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
